helm/cilium/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 0.16.0
+version: 1.13.6
```

Comment:
I think the `version` field will be overwritten by our automation when creating a new release, so it doesn't really mater which value we put in here now.

Reply:
I see the next version number in the create_release automation is coming from the latest release version. I'm not sure how sustainable this is, as it might create confusion every time the file is updated.

Reply:
Yeah, I think it's confusing, but I can't think of a better solution if we want to have this file copied from the other repo.

Reply:
Yeah, that's why I consider that we should not touch this file nor the `values.yaml` with **vendir**.
 
there is the option of using the ignorePath feature form vendir (like this https://github.com/giantswarm/ingress-nginx-app/blob/main/vendir.yml#L17) but I haven't tested it myself

Reply:
I'll keep the `Chart.yaml` for now. If in the future we see it's a problem, we can try to find a solution.